ci/pod/keycloak/kcadm_configure_university.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
```

Comment:
why we need this file?

Reply:
There are existed auth resource (or operation) create before in docker image `sshniro/keycloak-apisix:1.0.0`, which affects test cases in t/. 

So we need to record these operations in a script, then repeat them in unified CI Keycloak docker service.

Reply:
@moonming please help check this